include/flatbuffers/base.h

```diff
@@ -139,11 +139,21 @@
   #endif
 #endif // !defined(FLATBUFFERS_LITTLEENDIAN)
 
+// Current Version.
 #define FLATBUFFERS_VERSION_MAJOR 1
 #define FLATBUFFERS_VERSION_MINOR 11
-#define FLATBUFFERS_VERSION_REVISION 0
+#define FLATBUFFERS_VERSION_REVISION 1
```

Comment:
Java has gone to 1.11.1, the other languages really haven't, though I guess that's not a problem

Reply:
Removed.